src/Agent.Worker/CodeCoverage/CodeCoverageCommands.cs

```diff
@@ -270,6 +272,28 @@ private void LoadPublishCodeCoverageInputs(Dictionary<string, string> eventPrope
             }
         }
 
+        /// <summary>
+        /// This method replaces the default index.html generated by cobertura with 
+        /// the non-framed version
+        /// </summary>
+        /// <param name="reportDirectory"></param>
+        private void ModifyCoberturaIndexDotHTML(string reportDirectory, IExecutionContext executionContext)
+        {
+            try
+            {
+                if (_codeCoverageTool.Equals("cobertura", StringComparison.OrdinalIgnoreCase) && File.Exists(Path.Combine(reportDirectory, CodeCoverageConstants.DefaultIndexFile)) && File.Exists(Path.Combine(reportDirectory, CodeCoverageConstants.DefaultNonFrameFileCobertura)))
```

Comment:
suggest to use string.Equals(string1, string2, StringComparison.OrdinalIgnoreCase) as a practice